spec/lbvserver_spec.rb

```diff
@@ -52,7 +52,7 @@
     it 'when showing a particular lb vserver string is invalid' do
       expect {
         connection.lb.vserver.show('asdf')
-      }.should raise_error(TypeError, /convert/)
+      }.should raise_error(TypeError, /conver(t|sion)/)
```

Comment:
Ruby 2.0+ `TypeError` message in this case was `conversion` rather than `convert`. Updated this regex to check for both.